spec/listformat.html

```diff
@@ -62,7 +62,8 @@ <h1>CreatePartsFromList ( _listFormat_, _list_ )</h1>
         1. If _size_ is 0, then
           1. Return a new empty List.
         1. If _size_ is 2, then
-          1. Let _pattern_ be _listFormat_.[[TemplatePair]].
+          1. Let _n_ be an index into _listFormat_.[[Templates]] based on _listFormat_.[[Locale]], _list_[0], and _list_[1].
```

Comment:
Would the actual logic of selecting `n` based on these values be implementation-dependent here?

Reply:
Yes. Do you think it should say so more explicitly?

Reply:
Added "implementation-defined".

Reply:
Perfect! Thanks.